chapter2/tests/t12/nginx.conf

```diff
@@ -0,0 +1,74 @@
+user nginx nginx;
+worker_processes 1;
+error_log /tmp/camp/error.log notice;
+error_log /tmp/camp/error_debug.log debug;  
+pid /run/nginx.pid;
+
+events {
+	worker_connections 1024;
+}
+
+http {
+
+	log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+					  '$status $body_bytes_sent "$http_referer" '
+					  '"$http_user_agent" "$http_x_forwarded_for"';
+
+	access_log  /tmp/camp/access.log main;
```

Comment:
group directives by purpose. Your error_log is at the very top. Same should be for the access_log

Reply:
> group directives by purpose. Your error_log is at the very top. Same should be for the access_log

@dmitry-mightydevops 
Done